src/core/common/code_utils.hpp

```diff
@@ -98,22 +98,28 @@
         }                      \
     } while (false)
 
+/**
+ * Use this macro in conjunction with `VerifyOrExit()` when no action is specified.
+ *
+ */
+#define OT_NOOP
```

Comment:
```suggestion
#define Nop
```
`OT_NOOP` is always used together with `VerifyOrExit`. So, should it follow the same naming convention of `VerifyOrExit`?
Besides, `nop` is a common assembly instruction and should be familiar with embedded developers. 
FYI: https://en.wikipedia.org/wiki/NOP_(code)

Reply:
The wiki article shows "NOP", "no-op", and "NOOP" as all valid abbreviations. At the same time, "OT_NOOP" is not the same as it does not compile to anything.

As for uppercase vs camel-case, the OpenThread project generally uses all upper case for macros with no arguments.

I originally had this as "OT_NO_ACTION". Would you prefer that?

Reply:
I would prefer `OT_NOOP` to `OT_NO_ACTION`.